changelog/changed-paths.md

```diff
@@ -0,0 +1,3 @@
+The following are now available under `probe_rs::probe`: `AttachMethod`, `DebugProbe`, `DebugProbeError`, `DebugProbeInfo`, `DebugProbeSelector`, `ProbeCreationError`, `ProbeFactory`, `WireProtocol`
+`Probe` is now available under `probe_rs::Probe`
```

Comment:
```suggestion
`Probe` is now available under `probe_rs`
```

this would streamline with how the other changes were described :)

Reply:
typoed, type is under `probe_rs::probe` :(

Reply:
do we want to put this into the list on the line above then?